packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -38,47 +37,18 @@ sapien, et dapibus mi aliquet non. Pellentesque auctor sagittis lectus vitae rho
 ante in, vestibulum nulla.`;
 const richMessage = <span style={{ fontWeight: fontWeightBold }}>Successful alert with custom rendered message</span>;
 
-class BpkBannerDismissable extends Component {
-  constructor() {
-    super();
-
-    this.setDismissed = this.setDismissed.bind(this);
-
-    this.state = {
-      show: true,
-    };
-  }
-
-  setDismissed() {
-    this.setState({
-      show: false,
-    });
-  }
-
-  render() {
-    return (
-      <BpkBannerAlert
-        className={componentClassName}
-        message={this.props.message}
-        type={this.props.type}
-        dismissable
-        onDismiss={this.setDismissed}
-        show={this.state.show}
-        dismissButtonLabel="Dismiss"
-      />
-    );
-  }
-}
-
-BpkBannerDismissable.propTypes = {
-  message: PropTypes.string,
-  type: PropTypes.string,
-};
-
-BpkBannerDismissable.defaultProps = {
-  message: null,
-  type: null,
-};
+const BannerAlert = withDefaultProps(BpkBannerAlert, {
```

Comment:
Small documentation refactor, I'm using `withDefaultProps` to create three components so there's less repeated code :-)

Reply:
👍